### PR TITLE
Clean Command interface code

### PR DESCRIPTION
**Pull request by makiam**
Small Command interface usages clean.
default code for execute() method
useless super() calls removed
mark code with Override annotations

### CODE REVIEW

File: src/artofillusion/polymesh/Command.java [hunk 1]
```diff
@@ -22,7 +23,9 @@
      * execute the command
      *
      */
-    public void execute();
+    public default void execute() {
+      redo();
```
Comment (Sailsman63) [inline src/artofillusion/polymesh/Command.java:27]: I don't think this makes sense. I'd have to go through a lot of code to see why so many of the implementations of `execute()` just call `redo()`, but this is not an appropriate default.

**Review by Sailsman63 — CHANGES_REQUESTED** [review]

File: src/artofillusion/polymesh/Command.java (hunk 1 above)
Comment (makiam) [reply src/artofillusion/polymesh/Command.java:27]: I think all this PR has no sense without this change

Comment (Sailsman63) [reply src/artofillusion/polymesh/Command.java:27]: That may be. I'm trying to understand why the PME is using it's own undo/redo system here.

**Comment by Sailsman63** [comment]
I think we need a better idea of what the architecture is trying for. "Most common use" is not quite enough to justify a default implementation.

**Pull request closed without merging**